src/viam/utils.py

```diff
@@ -160,8 +159,12 @@ def datetime_to_timestamp(dt: datetime) -> Timestamp:
     return timestamp
 
 
-def container_to_list(container: RepeatedCompositeFieldContainer) -> List[Any]:
-    return [item for item in container]
+async def get_geometries(client, extra: Optional[Dict[str, Any]] = None, timeout: Optional[float] = None) -> List[Any]:
```

Comment:
can you create a protocol for a client that can get geometries like Stoppable and Reconfigurable?